homeassistant/components/light/wink.py

```diff
@@ -63,20 +71,48 @@ def available(self):
         """True if connection == True."""
         return self.wink.available
 
+    @property
+    def xy_color(self):
+        """Current bulb color in CIE 1931 (XY) color space."""
+        if not self.wink.supports_xy_color():
+            return None
+        return self.wink.color_xy()
+
+    @property
+    def color_temp(self):
+        """Current bulb color in degrees Kelvin."""
+        if not self.wink.supports_temperature():
+            return None
+        return color_util.color_temperature_kelvin_to_mired(
+            self.wink.color_temperature_kelvin())
+
     # pylint: disable=too-few-public-methods
     def turn_on(self, **kwargs):
         """Turn the switch on."""
         brightness = kwargs.get(ATTR_BRIGHTNESS)
+        rgb_color = kwargs.get(ATTR_RGB_COLOR)
+        color_temp_mired = kwargs.get(ATTR_COLOR_TEMP)
+
+        state_kwargs = {
+        }
+
+        if rgb_color:
+            state_kwargs['color_xy'] = color_util.color_RGB_to_xy(*rgb_color)
+
+        if color_temp_mired:
+            # HA returns mireds in reverse.  Flip it around.
```

Comment:
@balloob On that note.  Do you know if there was any reason the minimum of 154 and maximum of 500 were chosen?  

I feel like that range could, practically, be expanded to 800 and 100 because many color-flexible bulbs will operate in those ranges.